src/Codeception/Module/Yii2.php

```diff
@@ -288,6 +288,35 @@ public function amLoggedInAs($user)
      * ]);
      * ```
      *
+     * Note: if you need to load fixtures before the test (probably before the cleanup transaction is started;
+     * `cleanup` options is `true` by default), you can specify fixtures with _fixtures method of a testcase
+     * ```php
+     * <?php
+     * // inside Cest file or Codeception\TestCase\Unit
+     * public function _fixtures(){
+     *     return [
+     *         'user' => [
+     *             'class' => UserFixture::className(),
+     *             'dataFile' => codecept_data_dir() . 'user.php'
+     *         ]
+     *     ];
+     * }
+     * ```
+     * instead of
```

Comment:
"...instead of defining them in `_before`"

That's all. No need for code example below.